tests/demo/scene_largescene.hpp

```diff
@@ -0,0 +1,80 @@
+
```

Comment:
please remove this file. koen will have the same file and that one has lights placed on the sun temple candle thingies.